lib/matplotlib/tests/test_patches.py

```diff
@@ -503,7 +503,70 @@ def test_fancyarrow_units():
 
 @image_comparison(["large_arc.svg"], style="mpl20")
 def test_large_arc():
-    ax = plt.figure().add_subplot()
-    ax.set_axis_off()
-    # A large arc that crosses the axes view limits.
-    ax.add_patch(mpatches.Arc((-100, 0), 201, 201))
+    fig, (ax1, ax2) = plt.subplots(1, 2)
+    x = 210
+    y = -2115
+    diameter = 4261
+    for ax in [ax1, ax2]:
+        a = mpatches.Arc((x, y), diameter, diameter, lw=2, color='k')
+        ax.add_patch(a)
+        ax.set_axis_off()
+        ax.set_aspect('equal')
+    # force the high accuracy case
+    ax1.set_xlim(0, 20)
+    ax1.set_ylim(0, 20)
+
+    # force the low accuracy case
+    ax2.set_xlim(-23399, 18023)
+    ax2.set_ylim(-20044, 6584)
+
+
+@image_comparison(["rotated_arcs.svg"], style="mpl20")
```

Comment:
This test seems right, but it's rather difficult to say because the line widths are so large compared to the size of the `Arc`s.